jsdocConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "tags": {
+    "allowUnknownTags": true
+  },
+  "opts": {
+    "destination": "./docs"
```

Comment:
ok so just to clarify, move `docs` to `dist/docs`? anything else? Should we merge your Travis PR first? I'm a little unclear on next steps.